docs/5.0/preview/database-access.md

```diff
@@ -223,10 +428,9 @@ $ psql "service=postgres-aurora user=<db-user> database=<db-name>"
 Your browser does not support the video tag.
 </video>
 
-
 ## RFD
 
-Please refer to the [RFD document](https://github.com/gravitational/teleport/blob/roman/rfd/dba/rfd/0011-database-access.md)
+Please refer to the [RFD document](https://github.com/gravitational/teleport/blob/master/rfd/0011-database-access.md)
```

Comment:
We should link to the commit hash in case it gets renamed/moved.

```suggestion
Please refer to the [RFD document](https://github.com/gravitational/teleport/blob/024fd582e62560ffc925cd4b4d42c156043c041b/rfd/0011-database-access.md)
```